### PR TITLE
Remove long press on address in switching wallets

### PR DESCRIPTION
**Pull request by tchayen**
Fixes RNBW-4243
Figma link (if any):

## What changed (plus any additional context for devs)

Refer to this comment: [Linear](https://linear.app/rainbow/issue/RNBW-4243#comment-e8f42478) for current spec.

## Screen recordings / screenshots

Fix:

https://user-images.githubusercontent.com/5769281/183931860-255d3b92-b165-4590-969e-25235d6b4e9f.mp4

## What to test

See the attached video. Check wallet switching sheet. Try pressing, long pressing etc.

## Final checklist

- [x] Assigned individual reviewers?
- [x] Added labels? (`team1/team2`, `critical path`, `release`, `dev QA`)
- [x] Did you test both iOS and Android?
- [x] If your changes are visual, did you check both the light and dark themes?
- [x] Added e2e tests? If not, please specify why
- [x] If you added new critical path files, did you update the CODEOWNERS file?
- [x] If no `dev QA` label, did you add the PR to the QA Queue?

### CODE REVIEW

File: src/components/change-wallet/AddressRow.js
Comment (tchayen) [inline src/components/change-wallet/AddressRow.js:32]: If wallet doesn't have a name, it is shown with wrong font (default Roboto) on Android. This seems to fix the problem.

**Review by BrodyHughes — COMMENTED** [review]
Screenshot_20220810-104519

This looks great. Only question I have is: When a wallet has an ENS profile we aren't showing the emoji, but on this delete UI it still shows the emoji. Should we remove that for wallets that have a profile setup?

**Review by osdnk — APPROVED** [review]

**Comment by tchayen** [comment]
Blocked by design review. We are considering not enabling this again.

**Review by terrysahaidak — APPROVED** [review]

**Review by perunt — APPROVED** [review]

**Comment by tchayen** [comment]
Updated with changes from design review. Re-requested review from @perunt and @terrysahaidak. Added to QA queue again.

**Comment by tchayen** [comment]
Refer to this comment: [Linear](https://linear.app/rainbow/issue/RNBW-4243#comment-e8f42478) for current spec.

File: src/components/native-context-menu/contextMenu.android.js
Comment (tchayen) [inline src/components/native-context-menu/contextMenu.android.js:10]: This is not needed for this PR anymore, but it is supported by the lib so it might make some sense to leave it to highlight to devs that this is available. Not sure though.

**Review by terrysahaidak — APPROVED** [review]

**Review by estebanmino — COMMENTED** [review]
seems like the pressable area on iOS got smaller https://www.loom.com/share/b04c4b506cd540c2b8bde5d809083f48

on the video i'm trying to press the wallet name on the wallets sheet but the wallet doesn't change, when i press on the white area between the name and the ✅  icon works fine

working fine on android

**Comment by tchayen** [comment]
@estebanmino oops, sorry. I fixed it now.

There is one small issue remaining – on Android it's not possible to tap on the white space between name and edit button. But I think it's a small problem and we can live with it.

More details: if I wrap the whole row in the context menu then there's a problem with origin from which it shows up and it will look bad when user taps on the edit button and it shows up from the middle of the row. Nested context menus don't work because the outer one will take priority. The only thing would be to refactor the whole row to make the left side take up all remaining space. 

https://user-images.githubusercontent.com/5769281/186683736-2d3b1a68-7c58-4702-b2cf-6d435a317c6e.mov

**Review by estebanmino — APPROVED** [review]
🥇  https://www.loom.com/share/4d1208bc590f49c3b4591e728dbc1fc6

**Review by perunt — APPROVED** [review]

**Pull request merged**
